encoding/preparation.md

```diff
@@ -204,23 +204,14 @@ and which ones you want to use.
 
 The codec used most commonly is x264,
 an implementation of the h.264 standard.
-There are multiple versions available
-(kmod and tmod being the most popular ones),
-but they only differ slightly
-(mainly cosmetic patches and some obscure parameters
-that you will probably never use),
-and no version is objectively better than any other.
-x264 can encode in 8 or 10 bit color depth.
-We will explain the meaning of these at a later time.
-For now, they are just relevant for choosing the right binary.
-All modifications come in two versions each,
-one for 8 bit and one for 10 bit[^2].
-Most fansub group nowadays use 10 bit,
-so you should download that
-unless your group leader or encode mentor told you otherwise.
-
-- [Download kmod](http://komisar.gin.by/)
-- [Download tmod](https://github.com/jpsdr/x264/releases)
+The most recent builds can be found [here][x264vlan].
+Pick the most recent build for your operating system.
+At the time of writing this,
+win64's recent build is `x264-r2935-545de2f.exe`
+from 25-Sep-2018.
+(Notice: the 10-bit binaries are no longer separate from the 8-bit,
```

Comment:
Please add a date for the first release to include both output depths in the same binary.